float32 distance_reward
float32 velocity_reward
